src/Pulsar.Client/Internal/ProducerImpl.fs

```diff
@@ -593,6 +633,7 @@ type internal ProducerImpl<'T> private (producerConfig: ProducerConfiguration, c
     do mb.Error.Add(fun ex -> Log.Logger.LogCritical(ex, "{0} mailbox failure", prefix))
     do startSendTimeoutTimer()
     do startStatTimer()
+    do startCryptoTimer()
```

Comment:
Can we use MemoryCache like in java version instead of doing timer with mutable field?